code/modules/reagents/chemistry/reagents/blob_reagents.dm

```diff
@@ -592,75 +476,3 @@
 			for(var/mob/living/L in T)
 				L.adjust_fire_stacks(-2.5)
 				L.ExtinguishMob()
-
-//does brute damage and throws or pulls nearby objects at the target
-/datum/reagent/blob/dark_matter
```

Comment:
These were the only two original blob chems I liked

Please dont remove my codebabbies